Testing/SystemTests/lib/systemtests/systemtesting.py

```diff
@@ -303,7 +303,7 @@ def validateWorkspaces(self, valNames=None, mismatchName=None):
         checker.setPropertyValue("Workspace1", valNames[0])
         checker.setPropertyValue("Workspace2", valNames[1])
         checker.setProperty("Tolerance", float(self.tolerance))
-        if hasattr(self, 'tolerance_is_reller') and self.tolerance_is_reller:
+        if hasattr(self, 'tolerance_is_relerr') and self.tolerance_is_relerr:
```

Comment:
Since you are changing it anyhow, what about `self.tolerance_is_rel_err`?

Reply:
It would be even more in line with `CompareWorkspaces`, then. Will do.